packages/e2e/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@celo/e2e",
```

Comment:
should we mark as private? 
i saw its excluded from changeset so i assume its not ment to be published

Reply:
well it's not really a private package but sure we can do that too